### PR TITLE
Search FAB has been moved

### PR DESCRIPTION
**Pull request by pusztaienike**
We had to move the Search FAB from bottom-right cause of the chat app

### CODE REVIEW

File: apps/sensenet/src/localization/default.ts [hunk 1]
```diff
@@ -338,7 +339,7 @@ const values = {
     openInSearchTitle: 'Open in Search',
     openInSearchDescription: 'Opens the query expression in the Search view',
     saveQuery: 'Save Query',
-    newSearch: 'New search',
+
```
Comment (B3zo0) [inline apps/sensenet/src/localization/default.ts]:
```suggestion
```

File: apps/sensenet/src/components/SearchButton.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from 'react'
+import { useRepository } from '@sensenet/hooks-react'
```
Comment (B3zo0) [inline apps/sensenet/src/components/SearchButton.tsx]: Please follow the naming convention for files. We use kebab-case for new files now.

File: apps/sensenet/src/components/SearchButton.tsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+      },
+    },
+    navLinkListItem: {
+      width: '100%',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'space-evenly',
+      textDecoration: 'none',
+    },
+    listDropdown: {
```
Comment (B3zo0) [inline apps/sensenet/src/components/SearchButton.tsx]: This style and the next one is not used. Please remove these.

File: apps/sensenet/src/components/drawer/PermanentDrawer.tsx [hunk 4]
```diff
@@ -108,7 +109,12 @@ const PermanentDrawer: React.FunctionComponent<RouteComponentProps> = props => {
             ) : null}
```
Comment (B3zo0) [inline apps/sensenet/src/components/drawer/PermanentDrawer.tsx]: Can you also please get rid of the warning `Material-UI: the 'anchorEl' prop provided to the component is invalid.` by splitting the open and close button to have a separate Tooltip parent?

Comment (pusztaienike) [reply apps/sensenet/src/components/drawer/PermanentDrawer.tsx]: This solution won't solve this warning :(

**Pull request merged**